sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -408,6 +408,10 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     });
   }
 
+  async doesExist(guid: string): Promise<number> {
```

Comment:
I would recommend changing the return type to a boolean. A boolean value should fit better the question "doesExist".

Reply:
Modified the return type to boolean